src/components/sbb-form-field/sbb-form-field.tsx

```diff
@@ -431,6 +431,19 @@ export class SbbFormField implements ComponentInterface {
     await this._checkAndUpdateInputEmpty();
   }
 
+  /** Manually clears the input value. It only works for inputs, selects are not supported. */
+  @Method() public async clear(): Promise<void> {
+    (this._input as { value }).value = '';
```

Comment:
Not sure if we still should exit for other types than input? as a little safety mechanism :P

Reply:
Ok, sure